cellmlmanip/model.py

```diff
@@ -9,14 +10,18 @@
 from cellmlmanip.rdf import create_rdf_node
 from cellmlmanip.units import UnitStore
 
-
 logger = logging.getLogger(__name__)
 
-
 # Delimiter for variables name in Sympy expressions: <component><delimiter><name>
 SYMPY_SYMBOL_DELIMITER = '$'
 
 
+class DataDirectionFlow(Enum):
```

Comment:
I think I suggested `DataFlowDirection` which makes more sense to me than `DataDirectionFlow`. But happy to hear arguments the other way :)

Reply:
In fact to me `DataDirectionFlow` makes more sense but it doesn't really matter if you prefer the other. Maybe it's a male/female brain thing :-)

Reply:
OK, let's not bother changing it then :)